pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -199,26 +203,24 @@ let
       "zlib"
     ];
 
-    # Make sure Bazel knows about our configuration flags during fetching so that the
-    # relevant dependencies can be downloaded.
-    bazelFetchFlags = bazel-build.bazelBuildFlags;
-
-    bazelBuildFlags = [
```

Comment:
Why was this removed and merged into a single `bazelFlags`? It looks like we're losing `bazelFetchFlags` entirely?

Reply:
`bazelFlags` are inherited by both the build (https://github.com/NixOS/nixpkgs/blob/7b5ac19d7b5f6e029001b101cab7b73b7a45ed71/pkgs/build-support/build-bazel-package/default.nix#L55) and the fetch (https://github.com/NixOS/nixpkgs/blob/7b5ac19d7b5f6e029001b101cab7b73b7a45ed71/pkgs/build-support/build-bazel-package/default.nix#L59) derivations.

Reply:
Gotcha, thanks for explaining. In that case, could we keep that comment -- or something analogous -- explaining why we need those flags passed to both steps?